index.html

```diff
@@ -70,16 +68,113 @@ <h5>Skills</h5>
             </ul>
           </div>
         </div>
-
+        <hr />
         <div class="links">
           <a><i class="fa fa-github"></i></a>
           <a><i class="fa fa-twitter"></i></a>
           <a><i class="fa fa-envelope"></i></a>
           <a><i class="fa fa-linkedin"></i></a>
         </div>
+      </section>
+
+      <section class="works projects">
+        <span class="works1"></span>
+        <span class="works2"></span>
+        <div class="down">
+          <i class="fa fa-light fa-arrow-down"></i><span>Work</span>
+        </div>
+        <h3>MY RECENT</h3>
+        <h3>WORK</h3>
         <hr />
-        <p class="mail">joelkalema63@gmail.com</p>
+        <div class="project">
+          <div class="img">
+            <div class="picture"></div>
+          </div>
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div class="langage-used">
+            <ul>
+              <li>Css</li>
+              <li>Html</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button type="button">See Project</button>
```

Comment:
- [ ] Please notice that the "See project" button when hovered doesn't have the correct color. please use the correct color suggested in the Figma. that color should be `#5E42D7` when hovering over the button

|Your Design|Reference Design|
-|-
|![image](https://user-images.githubusercontent.com/18730106/153666439-037da915-8fff-4066-853f-6369561884af.png)|![image](https://user-images.githubusercontent.com/18730106/153666469-cac0e7f6-117f-49a1-b42e-1f82cef923a4.png)|

Reply:
about the button, I gave it a hover color but it has a normal color